docs/HowItWorks.md

```diff
@@ -69,3 +69,7 @@ rectors:
 
 - After this Rector displays list of changed files
 - Or with `--dry-run` option the diff of these files
+
+### Similar Projects
+
+- [ClangMR](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/41342.pdf) for C++ by Google (closed source) - almost idential workflow, develope independly though
```

Comment:
I wish we could see the real project. Would give us so many ideas :pray:

Reply:
I've added jscodeshift right under it. I saw the fluent API there as well.

Feel free to check it to dig some ideas